ui/resizable.js

```diff
@@ -1072,9 +1074,9 @@ $.ui.plugin.add("resizable", "ghost", {
 				margin: 0,
 				left: 0,
 				top: 0
-			})
-			.addClass("ui-resizable-ghost")
-			.addClass(typeof o.ghost === "string" ? o.ghost : "");
+			});
+		this._addClass( that.ghost, "ui-resizable-ghost",
+			typeof o.ghost === "string" ? o.ghost : "" );
```

Comment:
I know you just stayed with the existing logic, but this is silly. Don't call `._addClass()` if there's no class to add.